docs/01-Chart-Configuration.md

```diff
@@ -257,6 +257,18 @@ afterBody | `Array[tooltipItem], data` | Text to render after the body section
 beforeFooter | `Array[tooltipItem], data` | Text to render before the footer section
 footer | `Array[tooltipItem], data` | Text to render as the footer
 afterFooter | `Array[tooltipItem], data` | Text to render after the footer section
+dataPoints | `Array[CustomTooltipDataPoint], data` | List of matching point informations.
+
+#### CustomTooltipDataPoint
```

Comment:
I believe the section immediately below this describes the same thing. At a minimum we should add the new `x` and `y` properties to that code snippet as well. Maybe we should consider merging these sections as well.